### PR TITLE
MultiversX: Withdraw modal fixes

### PR DESCRIPTION
**Pull request by AlexandruPislariu**
### 📝 Description

MultiversX: Withdraw modal fixes

### ❓ Context

- **Impacted projects**: `LLC` `LLD` `LLM` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-8444

### ✅ Checklist

- [ ] **Test coverage** :x: Coins UI test coverage not possible just yet
- [x] **Atomic delivery** 
- [x] **No breaking changes**

### CODE REVIEW

**Review by haammar-ledger — COMMENTED** [review]
@AlexandruPislariu You're getting the `isGuarded` info from the network, but you don't seem to use it in the UI to display the warning or not. In fact I don't see that you're using this info at all. Is this expected?

**Comment by MiroMargineanu** [comment]
> @AlexandruPislariu You're getting the `isGuarded` info from the network, but you don't seem to use it in the UI to display the warning or not. In fact I don't see that you're using this info at all. Is this expected?

Hello @haammar-ledger! That was actually a mistake of mine. The variable wasn't used and instead a hard coded value was used a condition. I've now pushed though another commit fixing this problem and it should be fine. 👍

File: libs/ledger-live-common/src/families/elrond/api/dtos/multiversx-account.ts
[the remark this answers is not in the document]
Comment (AlexandruPislariu) [reply libs/ledger-live-common/src/families/elrond/api/dtos/multiversx-account.ts:1]: `dtos` stands for Data Transfer Objects.

[the remark this answers is not in the document]
Comment (haammar-ledger) [reply libs/ledger-live-common/src/families/elrond/api/dtos/multiversx-account.ts:1]: Haha you're right I forgot that. But in Live we rather use `types` as convention.

File: apps/ledger-live-desktop/src/renderer/families/elrond/AccountSubHeader.tsx [hunk 1]
```diff
@@ -29,42 +37,53 @@ const CustomButton = styled(Button)`
...
+      </Card>
 
-      <AccountSubHeaderDrawer
-        isOpen={isDrawerOpen}
-        closeDrawer={closeDrawer}
-        team="MultiversX"
-        family="MultiversX"
-      />
-    </Card>
+      {elrondResources.isGuarded && (
```
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/families/elrond/AccountSubHeader.tsx:81]: You need the same condition on the LLM part, right?

Comment (MiroMargineanu) [reply apps/ledger-live-desktop/src/renderer/families/elrond/AccountSubHeader.tsx:81]: Theoretically yes, but unfortunately we don't have access to the `account` object in the component where we could place it as far up top in the page as desired, so we will show the warning regardless of the guardian status, thus the variable isn't used. I've already discussed it [here](https://discord.com/channels/885256081289379850/922464539193278544/1132960610989129768). 👍

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
Code looks all good, please just check the CI, a couple of lint issues on LLC and LLD.

**Comment by MiroMargineanu** [comment]
> Code looks all good, please just check the CI, a couple of lint issues on LLC and LLD.

Hello, I've made now a push fixing the lint issue on mobile. There was only one, and on desktop I couldn't find any from our side. 👍

**Review by haammar-ledger — APPROVED** [review]
Pending validation from QA

**Pull request merged**